### PR TITLE
SC-7461 - tabbar shadow underline

### PR DESCRIPTION
**Pull request by xavero-evandro**
# Description
- The shadow underneath the tabbar is missing. there is a 3dp shadow underneath.
 - Make sure this shadow goes right to the end of the screen
## Links to Tickets or other pull requests

https://ticketsystem.hpi-schul-cloud.org/browse/SC-7461

## Changes

Added a underline shadow to the whole tabbar

## New Repos, NPM pakages or vendor scripts
Updated @schul-cloud/styles to 0.2.3

## Screenshots of UI changes

5-molecules-tabs-tabs_1-snap

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by xavero-evandro** [comment]
@schul-cloud-bot update screenshots

File: .dccache [hunk 1]
```diff
@@ -0,0 +1 @@
+[{"/Users/xavero/Projects/nuxt-client/src/components/organisms/Tabs/Tabs.vue":"1"},[3651,1603811580967.4438,"2"],"bf8c0d49e61288037feac7e6a67c088e9c360dd580f8c9f194c0c61018df39d5"]
```
Comment (haniapalac) [inline .dccache:1]: What is this file? Or can we remove it?

File: src/components/organisms/Tabs/Tabs.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-	<div>
+	<div class="test">
```
Comment (haniapalac) [inline src/components/organisms/Tabs/Tabs.vue:2]: Why is the class named test? Or can the name be more connected to what the component does/this wrapper is about?

File: src/components/organisms/Tabs/Tabs.vue [hunk 3]
```diff
@@ -64,10 +64,11 @@ export default {
 }
 ul.tabs {
 	display: flex;
-	justify-content: space-between;
+	justify-content: center;
 	max-width: 1024px;
 	padding: 0;
 	margin-bottom: var(--space-md);
+	// border-bottom: 2px solid var(--color-gray-light);
```
Comment (haniapalac) [inline src/components/organisms/Tabs/Tabs.vue:71]: I guess we can remove this line altogether.

File: src/components/organisms/Tabs/Tabs.vue [hunk 4]
```diff
@@ -103,39 +104,55 @@ ul.tabs {
...
+				color: var(--color-gray-dark);
+			}
 		}
 	}
+	&::after {
+		position: absolute;
+		top: calc(4.45 * (var(--space-md)));
+		z-index: var(--layer-behind);
+		width: 100vw;
+		height: 2px;
```
Comment (haniapalac) [inline src/components/organisms/Tabs/Tabs.vue:133]: Maybe this could be changed to relative units?

File: src/components/organisms/Tabs/Tabs.vue [hunk 5]
```diff
@@ -103,39 +104,55 @@ ul.tabs {
...
+		content: " ";
+		background: var(--color-gray-light);
+		box-shadow: 0 0 1px 0 var(--color-gray-light);
+	}
 }
 
-@media screen and (max-width: 450px) {
+@media (max-width: 568px) {
 	ul.tabs {
 		max-width: 450px;
```
Comment (haniapalac) [inline src/components/organisms/Tabs/Tabs.vue:142]: Maybe we can have relative units here too?

File: src/components/organisms/Tabs/Tabs.vue [hunk 6]
```diff
@@ -144,7 +161,7 @@ ul.tabs {
 	}
 }
 
-@media screen and (max-width: 1112px) and (orientation: landscape) {
+@media (max-width: 1112px) and (orientation: landscape) {
 	ul.tabs {
 		max-width: 1023px;
```
Comment (haniapalac) [inline src/components/organisms/Tabs/Tabs.vue:166]: Maybe relative units here too?

**Review by janrenz — DISMISSED** [review]
the dccache file still seems present in the PR

**Comment by xavero-evandro** [comment]
@schul-cloud-bot update screenshots

File: src/components/organisms/Tabs/Tabs.vue [hunk 7]
```diff
@@ -59,13 +59,18 @@ export default {
...
+		margin-right: var(--space-xs-3);
+		font-size: var(--text-md);
+	}
 }
 ul.tabs {
 	display: flex;
-	justify-content: space-between;
-	max-width: 1024px;
+	justify-content: center;
+	max-width: 450px;
```
Comment (haniapalac) [inline src/components/organisms/Tabs/Tabs.vue:73]: For now you can use local variables if we can't update the styles library. Shouldn't we also use rescalable units like rems?

File: src/components/organisms/Tabs/Tabs.vue [hunk 8]
```diff
@@ -96,56 +101,59 @@ ul.tabs {
...
-			display: none;
-			animation: fadeEffect var(--duration-transition-medium) ease-in-out;
-		}
-	}
-}
 
-@media screen and (max-width: 1112px) and (orientation: landscape) {
-	ul.tabs {
+	@include breakpoint(tablet) {
 		max-width: 1023px;
```
Comment (haniapalac) [inline src/components/organisms/Tabs/Tabs.vue:157]: Here as well we should use rescaleable units.

File: src/components/organisms/Tabs/Tabs.vue (hunk 7 above)
Comment (haniapalac) [inline src/components/organisms/Tabs/Tabs.vue:73]: use rems or vw etc.

**Review by haniapalac — CHANGES_REQUESTED** [review]

**Review by haniapalac — DISMISSED** [review]

**Comment by xavero-evandro** [comment]
@schul-cloud-bot update screenshots

**Comment by xavero-evandro** [comment]
@schul-cloud-bot update screenshots

**Review by haniapalac — DISMISSED** [review]

**Comment by xavero-evandro** [comment]
@schul-could-bot update screenshots

**Review by haniapalac — APPROVED** [review]

**Comment by xavero-evandro** [comment]
@schul-cloud-bot update screenshots

**Pull request merged**